langfuse/openai.py

```diff
@@ -170,6 +170,17 @@ def get_langfuse_args(self):
         return {**self.args, **self.kwargs}
 
     def get_openai_args(self):
+        # OpenAI returns streaming usage not by default but only if stream_options has include_usage set
+        if self.kwargs.get("stream") and "stream_options" not in self.kwargs:
```

Comment:
adding this automatically adds an empty final "choices" chunk and consumer code might not handle that, so that's a breaking change :grimacing: